packages/cfd/src/Helpers/config.tsx

```diff
@@ -103,9 +103,11 @@ export const PlatformsDesktopDownload = ({
                     }
                 }}
             >
-                <Text color='general' size='xxs' weight='bold'>
-                    <Localize i18n_default_text='Web terminal' />
-                </Text>
+                <div className='cfd-trade-modal__platform-button-text'>
```

Comment:
Text component can accept `className` props

Reply:
which we don't need since the props (color, size, weight) are doing what we need